README.md

```diff
@@ -119,8 +119,35 @@ collections of "utility functions" nor for packages that are only useful for a c
 for example [LocalRegistry.jl](https://github.com/GunnarFarneback/LocalRegistry.jl). The
 [Pkg documentation about registries](https://pkgdocs.julialang.org/v1/registries/) might be useful
 if you decide to go this route.
+* Packages that are ["vibe-coded"](https://simonwillison.net/2025/Mar/19/vibe-coding/)
+(generated by an LLM without human review) are not suitable for registration. See the
+[LLM policy](#is-there-any-policy-regarding-the-use-of-llms-in-registered-packages).
 * "Empty" packages that do not yet have functionality are not ready to be registered.
 
+### Is there any policy regarding the use of LLM's in registered packages?
+
+It is perfectly fine to register packages that have been produced with the
+[_assistance_](https://discourse.julialang.org/t/the-use-of-claude-code-in-sciml-repos/131009/8)
+of a large language model (LLM) such as Claude Code or any similar
+tool. However, it is essential that a human maintainer has a full
+understanding of the generated code.
+
+[It is not okay to submit vibe-coded packages](https://discourse.julialang.org/t/should-general-have-a-guideline-or-rule-preventing-registration-of-vibe-coded-packages/133205),
+defined as "building software with an LLM without reviewing the code it
+writes". Packages that exhibit obvious signs of ["AI slop"](https://en.wikipedia.org/wiki/AI_slop)
+may be blocked from registration. To ensure that a package containing
+LLM-generated components is suitable for this registry, follow best
+practices as much as possible:
+
+* Review and understand all generated code, by hand.
```

Comment:
let's add Keno's disclosure suggistion (https://discourse.julialang.org/t/should-general-have-a-guideline-or-rule-preventing-registration-of-vibe-coded-packages/133205/16?u=ericphanson). Wording from [base](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#contributor-checklist)
```suggestion
* if your package contains substantial contributions from a generative AI tool, please disclose so with details in the README
* Review and understand all generated code, by hand.
```